ament_cppcheck/bin/ament_cppcheck

```diff
@@ -41,7 +42,11 @@ def main(argv=sys.argv[1:]):
         print('No files found', file=sys.stderr)
         return 1
 
-    cppcheck_bin = find_executable('cppcheck')
+    additional_paths = None
+    if os.name == 'nt':
+        # search in location where cppcheck is installed via chocolatey
+        additional_paths = ['C:/Program Files (x86)/Cppcheck']
```

Comment:
The problem is that I don't think that will get expanded by the `which` command, instead you'll need to get `program_file = os.environ.get('ProgramFiles(x86)', "C:/Program Files (x86)/")` and use that in the extra path.